mopidy/backend.py

```diff
@@ -31,38 +71,40 @@ class Backend:
     #: The library provider. An instance of
     #: :class:`~mopidy.backend.LibraryProvider`, or :class:`None` if
     #: the backend doesn't provide a library.
-    library = None
+    library: Optional["LibraryProvider"] = None
```

Comment:
Could the above
```
audio = None
```
be
```
audio: Optional[Audio] = None
```
like in `PlaybackProvider.__init__()`?

Reply:
I'll change it to `Optional[Any]` for now, with a todo of replacing `Any` with something like `ActorProxy[Audio]` when we've figured out the best way to model Pykka actors.